client/src/components/QuestionAnswerPart/QACard.jsx

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Moment from 'react-moment';
+import QuestionModal from './QuestionModal.jsx';
+import HelpfulNess from './HelpfulNess.jsx';
+import AnswerModal from './AnswerModal.jsx';
+
+const configobj = require('../../../../config.js');
+
+const Container = styled('div')`
+  display:flex;
+  direction:row;
+  flex-wrap:wrap
+`;
+
+const Button = styled('button')`
+  border:none;
+  background:none;
+  text-decoration:underline;
+  cursor:pointer;
+`;
+const url = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/';
+export default function QandA({ ele, key, productName }) {
+  const answerslist = Object.values(ele.answers).sort((a, b) => {
+    if (a.answerer_name.toLowerCase() === 'seller') {
+      return -1;
+    } if (b.answerer_name.toLowerCase() === 'seller') {
+      return 1;
+    }
+    return (b.helpfulness - a.helpfulness);
+  });
+  const [showModal, setShowModal] = useState(false);
+  const [questionId, setQuestionId] = useState(0);
+  const [questionText, setQuestionText] = useState('');
+  const [report, setReport] = useState(false);
+  const [displayedAns, setDisplayedAns] = useState(answerslist.slice(0, 2));
+  const [showCollapseBtn, setShowCollapeBtn] = useState(false);
+  const [showLoadMoreBtn, setShowLoadMoreBtn] = useState(true);
```

Comment:
Is this state still needed? I saw you have something similar in QuestionsAndAnswers.jsx